Jenkinsfile

```diff
@@ -39,12 +40,13 @@ if ( config.branches.containsKey(env.BRANCH_NAME) ) {
                 }
             }
             utils.ircNotification(config, [stage: 'Test & Deploy', status: 'starting'])
-            try {
-                utils.buildDockerImage(dockerfile: 'bedrock_base', update: true)
-                utils.buildDockerImage(dockerfile: 'bedrock_code', fromDockerfile: 'bedrock_base')
-            } catch(err) {
-                utils.ircNotification(config, [stage: 'Docker Build', status: 'failure'])
-                throw err
+            lock ("bedrock-docker-${env.GIT_COMMIT}") {
+                try {
+                    sh 'docker/jenkins/build_images.sh --prod --test'
```

Comment:
yes. what those switches really mean are "build the prod code image" and "build the testing image", which are separate things. There's also a `--demo` flag that will build the image suitable for deploying to a demo having synced all the external data to a squlite db.